usr/src/uts/armv8/rpi4/bcm2711-emmc2/Makefile

```diff
@@ -34,7 +34,7 @@ UTSBASE	= ../../..
 MODULE		= bcm2711-emmc2
 OBJECTS		= $(BCM2711_EMMC2_OBJS:%=$(OBJS_DIR)/%)
 ROOTMODULE	= $(ROOT_RPI4_DRV_DIR)/$(MODULE)
-CONF_SRCDIR	= $(UTSBASE)/armv8/rpi4/io/$(MODULE)/
+CONF_SRCDIR	= $(UTSBASE)/armv8/rpi4/io
```

Comment:
We prefer the conf file and the driver be in the same directory.  Despite apparently sun4v not having done that :)

Reply:
It's probably not obvious in the diffs, but the drivers are now a single C file each and have moved into `armv8/rpi4/io` directly:
```
michael@omnios:/build/arm64-gate/illumos-gate/usr/src/uts$ ls -1 armv8/rpi4/io
bcm2711-emmc2.c
bcm2711-emmc2.conf
bcm2711-genet.c
bcm2711-genet.conf
michael@omnios:/build/arm64-gate/illumos-gate/usr/src/uts$
```

Does this look OK?

Reply:
I've been trying to do one-directory-per-driver (because I have big illumos build changes that depend upon it), but yes this is fine right now

Reply:
> I've been trying to do one-directory-per-driver (because I have big illumos build changes that depend upon it), but yes this is fine right now

I can change this back easily enough - I'm having to rename bcm2711-genet to bcm2711_genet anyway, so I'm renaming bcm2711-emmc2 to bcm2711_emmctwo at the same time. I'll update to go with one per directory.